APTITUDE/APTITUDETEST.Core/Validations/DateMustBeValidAttribute.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+
+namespace AptitudeTest.Core.Validations
+{
+    public class DateMustBeValidAttribute:ValidationAttribute
+    {
+        protected override ValidationResult? IsValid(object? value, ValidationContext validationContext)
+        {
+            var dateProperty = validationContext.ObjectType.GetProperty("Date");
+            
+            var date = (DateTime?)dateProperty.GetValue(validationContext.ObjectInstance);
+
+            if (date < DateTime.UtcNow)
+            {
+                return new ValidationResult("Date must be valid");
```

Comment:
put message in static file

Reply:
Due to circular dependency I can not use Response messages here